modules/bitgo/example/js/btc/make-lightning-payment.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Make lightning payment to a generated invoice.
+ *
+ * Copyright 2022 BitGo, Inc.  All Rights Reserved.
+ */
+const BitGoJS = require('bitgo');
+
+// change this to env: 'production' when you are ready for production
+const env = 'test';
+// change coin to 'btc' when working with production
+const coin = 'tbtc';
+
+const bitgo = new BitGoJS.BitGo({ env });
+
+// set your access token here
+const accessToken = '';
+
+// set your wallet from the YYYYY parameter here in the URL on app.bitgo-test.com
+// https://test.bitgo.com/enterprise/XXXXXXXXX/coin/tbtc/YYYYY/transactions
+const walletId = '';
+
+async function makeLightningPayment() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+  const wallet = await bitgo.coin(coin).wallets().get({ id: walletId });
+  const lightning = wallet.lightning();
+
+  const initialLightningBalance = await lightning.getBalance();
+  console.log('Initial Lightning Balance:', initialLightningBalance.balance);
+
+  await bitgo.unlock({ otp: '000000', duration: 3600 });
+  const walletInvoice = await lightning.createInvoice({ value: 1000, memo: 'Test Payment' });
```

Comment:
In this case the wallet would be paying itself, which we could do but I think might be confusing. Maybe just declare something like

```
const invoiceToPay = ''; // set the invoice you want to pay here
```